packages/homepage/content/docs/3-configuration.md

```diff
@@ -35,3 +35,5 @@ support these options:
 | `infiniteLoopProtection` | Whether we should throw an error if we detect an infinite loop                                    | `true`/`false`                                                                                                     | `true`                                             |
 | `hardReloadOnChange`     | Whether we should refresh the sandbox page on every change, good for sandboxes with global state. | `true`/`false`                                                                                                     | `false`                                            |
 | `template`               | Which sandbox template to use                                                                     | [see here](https://github.com/codesandbox-app/codesandbox-importers/blob/master/packages/types/index.d.ts#L24-L39) | smart detection, w/ fallback to `create-react-app` |
+| `container`              | Options inside the container object to                                                            |                                                                                                                    |                                                    |
```

Comment:
'Options inside the container object to' - to what? I don't understand this description, can we re-word?

Reply:
That was terrible english

Reply:
What I meant was these options are inside the container object